src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
```

Comment:
If I understood the code correctly, this checks if the car sees a tag or not. If it does not see a tag, then it floors the vehicle (by overriding a private variable). It does not pan the camera or do anything of that sort.